OntologyMapper/src/Mappings/Mapped/Json/v0/Willow/mapped_json_v0_dtdlv2_willow.json

```diff
@@ -4986,7 +4986,7 @@
     {
       "InputDtmi": "dtmi:mapped:core:TestPoint;1",
       // Mapped to Base - Review needed
```

Comment:
Is the review comment no longer needed?

Reply:
Still needed. Need to verify that TestPoint is the same as Point (it may not be)